section-05-production-model-package/MANIFEST.in

```diff
@@ -0,0 +1,18 @@
+include *.txt
```

Comment:
I thought we would not package the model in this section, and we would only focus on the ins and outs of writing production code.

Reply:
@solegalli Thanks for all the comments:

1. We will not be cover **publishing** in this section, but we still need to cover packaging (it doesn't fit anywhere near as neatly into the next section)

2. I guess not, I just hate manual steps...but ok fine we can remove (until the CI/CD section)

3. ok, will add the test

4. ok, will add a prediction value to check against (maybe use Math.isclose)